native/cocos/renderer/gfx-vulkan/VKCommands.cpp

```diff
@@ -159,7 +159,6 @@ void cmdFuncCCVKCreateTexture(CCVKDevice *device, CCVKGPUTexture *gpuTexture) {
                                              pVkImage, pVmaAllocation, &res);
             if (!result) {
                 gpuTexture->memoryAllocated = false;
-                return;
```

Comment:
Here we use TBR memoryless texture (memory on chip, or subpass attachment without load/store) and do not allocate memory. I'm not sure the change is correct or not.

Reply:
i got it~ yes this is a point. maybe we can have some strategy pattern or macro or whatever to make a improve on this? like, on WIN, it's just fine with the 'retry' way.
anyway i have just reverted this change, now about other 2 changes i think it's safe to be merged.